lib/commands/deploynodeapp.js

```diff
@@ -495,14 +696,29 @@ function handleUploadResult(err, req, fileName, itemDone) {
 
 // Create a target endpoint that references the Node.js script
 function createTarget(opts, request, done) {
-  var targetDoc = mustache.render(
+  var targetDoc;
+  
+  // we need the XML for a HostedTarget (or whatever)...this is just a place holder that isn't a ScriptTarget
+  if (opts.runtime === EdgeServicesRuntimeOption) {
+    targetDoc = mustache.render(
+    '<TargetEndpoint name="default">' +
```

Comment:
Won't we need the policy to get the routing key from the KVM? Not sure if this is the right place.